app/uk/gov/hmrc/essttpstubs/services/AffordabilityService.scala

```diff
@@ -38,26 +40,26 @@ class AffordabilityService @Inject() (config: Configuration) {
   private val totalInterestRate: BigDecimal = baseInterestRate + additionalInterestRate
 
   def calculateInstalmentAmounts(request: InstalmentAmountRequest): Either[Error, InstalmentAmounts] = {
-    val totalDebtAmount = request.debtItemCharges.map(_.outstandingDebtAmount.value).sum
-    val initialPaymentAmount = request.initialPaymentAmount.map(_.value).getOrElse(0L)
+    val totalDebtAmount: Long = request.debtItemCharges.map(_.outstandingDebtAmount.value.value).sum
+    val initialPaymentAmount: Long = request.initialPaymentAmount.map(_.value).getOrElse(0L)
     if (totalDebtAmount <= initialPaymentAmount)
       Left(BadRequestError("Total debt amount was less than or equal to the initial payment amount"))
-    else if (request.minPlanLength.value > request.maxPlanLength.value)
+    else if (request.paymentPlanMinLength.value > request.paymentPlanMaxLength.value)
       Left(BadRequestError("Min plan length was strictly greater than the max plan length"))
     else {
       val residualDebtAmount = BigDecimal(totalDebtAmount - initialPaymentAmount)
       val interestPerMonth = totalInterestRate * residualDebtAmount / BigDecimal(1200)
 
-      val minInterest = request.minPlanLength.value * interestPerMonth
-      val maxInterest = request.maxPlanLength.value * interestPerMonth
+      val minInterest = request.paymentPlanMinLength.value * interestPerMonth
+      val maxInterest = request.paymentPlanMaxLength.value * interestPerMonth
 
-      val minInstalmentAmount = residualDebtAmount / request.maxPlanLength.value + maxInterest
-      val maxInstalmentAmount = residualDebtAmount / request.minPlanLength.value + minInterest
+      val minInstalmentAmount = residualDebtAmount / request.paymentPlanMaxLength.value + maxInterest
+      val maxInstalmentAmount = residualDebtAmount / request.paymentPlanMinLength.value + minInterest
 
       for {
         min <- toLong(minInstalmentAmount)
         max <- toLong(maxInstalmentAmount)
-      } yield InstalmentAmounts(AmountInPence(min), AmountInPence(max))
+      } yield InstalmentAmounts(ProcessingDateTime(LocalDate.now().toString), AmountInPence(min), AmountInPence(max))
```

Comment:
shouldn't this be a date time rather than a date time? I think if you use something like `LocalDateTime.now().toInstant(ZoneOffset.of("Z"))` you should get something which looks like the API would give